data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/filter_adjacent_inactive.cfg

```diff
@@ -0,0 +1,51 @@
+#textdomain wesnoth-test
+
+#####
+# API(s) being tested: ability[filter_adjacent]adjacent,count=
+##
+# Actions:
+# Give Alice an ability specialX, which is only active if one to six of adjacents units are bob.
```

Comment:
This looks like it's filtering for Loki rather than bob for some reason?

Reply:
Also the ability description below doesn't match this either.

Reply:
description corrected, Loki filterd nstead of bob because don't exist in map